src/components/Layout.js

```diff
@@ -41,7 +46,11 @@ export default function Layout() {
                 <a href='#hot'><li>Hot Topics</li></a>
                 <a href='#contact'><li>Contact</li></a>
                 <a href='#ref'><li>References</li></a>
-                <a onClick={onClickLoginShowToggle} href='#'><li>Login</li></a>
+                <a  
+                    href='#'
+                    onClick={onClickLoginShowToggle}
```

Comment:
it is a better practice to use `handleEventName` or `handleFunctionName`. In this case, it would be more understandable if it is renamed as`handleLoginToggle`